requirements.txt

```diff
@@ -23,3 +23,4 @@ dnspython==2.4.2
 asyncssh==2.14.1
 aioftp~=0.20.0
 packaging==23.2
+croniter~=2.0.5
```

Comment:
It looks like a new `croniter` major release (version `3.0.3`) came out since the PR was opened - can you confirm whether or not this feature still works with the upgrade?

Reply:
FWIW, it looks like it's working on my local test installation with version `3.0.3`